azurelinuxagent/common/exception.py

```diff
@@ -210,6 +210,28 @@ def __init__(self, msg=None, inner=None):
         super(ResourceGoneError, self).__init__(msg, inner)
 
 
+class RemoteAccessError(AgentError):
```

Comment:
seems like you need to update your branch

Reply:
There's still something funny with your branch. This exception was removed by #1935

Reply:
Yeah I dont think the merge happened properly/correctly. Even Travis is failing with some weird error. I'll try fixing it today properly if I get time. Will let you know once its ready for review!

Reply:
Not sure what went wrong here, I tried the merge again but somehow this is still there. I'll just cherry-pick that commit into my branch to get this change in. Apart from this I dont see anything out of ordinary